src/avitab/apps/ChartsApp.cpp

```diff
@@ -161,6 +178,7 @@ void ChartsApp::createPdfTab(const std::string &pdfPath) {
     auto page = tab.page;
     tab.window->setOnClose([this, page] {
         api().executeLater([this, page] {
+            settingsContainer->setVisible(false);

```

Comment:
There was a missing check if the container was actually created, causing a segfault if not - fixed during the merge.